contrib/go/src/python/pants/contrib/go/rules/targets.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.engine.target import (
```

Comment:
Big fan of this way of defining targets BTW, great work with the API

Reply:
Thanks :D